README.md

```diff
@@ -499,7 +499,7 @@ Programmer 2 [reviews the pull request](https://help.github.com/articles/about-p
 
 
 <!-- ***********************************************************-->
-## That's it 😄 Thank you for checking out the 'Git Workflow Workshop for Two Programmers' :clap:
+## That's it 😄 Thank you for checking out the 'Git Workflow Workshop for Two Programmers' :clap: A summary of the above commands and what they do can be found [here](/images/git-flow-summary-table.png) in a neat little table.
```

Comment:
Good idea to add those :smile: 
though I think the summery should be on a different line so it won't be a big chunk of text

maybe like this:

<!-- ***********************************************************-->
## That's it 😄 Thank you for checking out the 'Git Workflow Workshop for Two Programmers' :clap:  

***A summary of the above commands and what they do can be found [here](/images/git-flow-summary-table.png) in a neat little table.***

---

```
<!-- ***********************************************************-->
## That's it 😄 Thank you for checking out the 'Git Workflow Workshop for Two Programmers' :clap:  

***A summary of the above commands and what they do can be found [here](/images/git-flow-summary-table.png) in a neat little table.***

---
```

Reply:
@Amirk390  @eliascodes Sorry I didn't see this until now! Gonna change it 👍